server/src/main/java/org/candlepin/resource/ConsumerTypeResource.java

```diff
@@ -61,7 +61,8 @@ public ConsumerTypeResource(ConsumerTypeCurator consumerTypeCurator, I18n i18n)
         this.i18n = i18n;
     }
 
-    @ApiOperation(notes = "Retrieves a list of Consumer Types", value = "list")
+    @ApiOperation(notes = "Retrieves a list of Consumer Types", value = "list",
+        response = ConsumerType.class, responseContainer = "list")
```

Comment:
So it looks like the missing piece was the "response = <target class>.class, responseContainer = "list". If that's what we needed, what is the @Wrapped annotation doing here? Looks like the intent was the same.